src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -176,9 +176,20 @@ private boolean isVisibilitySetToAnonymousToRecipientAndInstructors(FeedbackQues
     }
 
     private boolean isVisibilitySetToAnonymousToRecipientVisibleToInstructors(FeedbackQuestionAttributes question) {
-        boolean responsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
-                && question.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS)
-                && question.showResponsesTo.contains(FeedbackParticipantType.RECEIVER);
+        boolean responsesVisibleOnlyToRecipientAndInstructors;
+
+        if (question.questionType.equals(FeedbackQuestionType.CONTRIB)) {
```

Comment:
If we don't want to create a new common feedback visibility option, we can just disable the common visibility options like Prof Damith says for `CONTRIB` type question and enforce the constrains using JS code for custom visibility settings.